server/jetstream_cluster_test.go

```diff
@@ -5386,7 +5389,7 @@ func TestJetStreamClusterAckPendingWithMaxRedelivered(t *testing.T) {
 	}
 
 	// Send in 100 messages.
-	msg, toSend := make([]byte, 32), 100
+	msg, toSend := []byte("HELLO"), 100
 	rand.Read(msg)
```

Comment:
Is that necessary?

Reply:
Not as it is but when I was debugging and printing things out it was much easier.

Reply:
Does not affect the test.